pages/account/statistics.js

```diff
@@ -58,16 +60,34 @@ export async function getServerSideProps(context) {
       }
     );
     data = await res.json();
+
+    let profileSections = [
```

Comment:
I think this doesn't need to be in the `try/catch`

Reply:
Moved it out of the try catch